docker/ci/ubuntu/Dockerfile

```diff
@@ -23,4 +23,6 @@ ADD $TARGETARCH/$TARGET_BIN /greptime/bin/
 
 ENV PATH /greptime/bin/:$PATH
 
+ENV TARGET_BIN=$TARGET_BIN
+
 ENTRYPOINT ["$TARGET_BIN"]
```

Comment:
The `ENTRYPOINT` cannot use environment variables [by design](https://github.com/moby/moby/issues/18492). The easiest way to fix it to use `sh -c exec`, for example:

```
ENTRYPOINT ["sh", "-c", "exec $TARGET_BIN \"$@\"", "--"]
```

Reply:
done